packages/react-instantsearch/src/core/highlight.test.js

```diff
@@ -1,101 +1,112 @@
 /* eslint-env jest, jasmine */
-import React from 'react';
+import {parseAlgoliaHit, parseHighlightedAttribute} from './highlight.js';
```

Comment:
Since only parseAlgoliaHit is actually used externally by other files, should we export it from the file and test it in isolation? Making it an export can induce the reader in error by thinking this is actually used by other modules while it's not.

I would go for:
- export default function highlight
- only test highlight()

What do you think?